include/kodlab_mjbots_sdk/joint_moteus.h

```diff
@@ -103,129 +114,193 @@ class JointMoteus: public JointBase
             float pos_min = -std::numeric_limits<float>::infinity(),
             float pos_max = std::numeric_limits<float>::infinity(),
             float soft_start_duration_ms = 1,
+            ::mjbots::moteus::QueryCommand query_type = kDefaultQuery,
             float moteus_kp = 0,
-            float moteus_kd = 0)
-            : JointBase("", direction, zero_offset, gear_ratio, max_torque, pos_min, pos_max, soft_start_duration_ms),
-              can_bus_(can_bus),
-              can_id_(can_id),
-              moteus_kp_(moteus_kp),
-              moteus_kd_(moteus_kd) {}
-
+            float moteus_kd = 0
+        );
 
         /**
          * @brief Construct a new Joint Moteus object with a MoteusJointConfig struct
          * 
          * @param config MoteusJointConfig input
          */
-        JointMoteus(MoteusJointConfig config)
-            : JointBase( config.name,
-                         config.direction,
-                         config.zero_offset,
-                         config.gear_ratio,
-                         config.max_torque,
-                         config.pos_min,
-                         config.pos_max,
-                         config.soft_start_duration_ms),
-              can_id_( config.can_id),
-              can_bus_( config.can_bus),
-              moteus_kp_(config.moteus_kp),
-              moteus_kd_(config.moteus_kd){}
+        JointMoteus(MoteusJointConfig config);
         
+        /**
+         * @brief Update the joint of the moteus. Converts rot/s to rad/s and saves other state values.
+         * 
+         * @param reply_message reply message sent by moteus to pi3hat
+         */
+        void UpdateMoteus(::mjbots::moteus::QueryResult reply_message);
+
         /**
          * @brief Destroy the JointMoteus object, virtual to ensure proper 
          * destruction of derived classes
          */
         virtual ~JointMoteus(){}
 
         /**
-         * @brief Update the joint of the moteus. Converts rot/s to rad/s and saves mode
+         * @brief Determines whether open loop options should be enabled
```

Comment:
Is this function calculating if open loop, or just returning a bool? If its calculating, what determines if the moteus is open loop? Make that clear in the comment

Reply:
Changed comment and function name to better describe the utility of this function. Open to additional changes/suggestions.